src/fsharp/Fsc/fsc.exe.config

```diff
@@ -13,6 +13,15 @@
           oldVersion="2.0.0.0-4.4.1.0"
           newVersion="4.4.1.0"/>
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity
+          name="System.ValueTuple"
+          publicKeyToken="b03f5f7f11d50a3a"
+          culture="neutral"/>
+        <bindingRedirect
+          oldVersion="2.0.0.0-4.4.1.0"
```

Comment:
I don't think value tuple needs this binding redirect.  In the future it may need one.  but not right now.